packages/vendure-plugin-invoices/src/api/invoice.service.ts

```diff
@@ -125,7 +118,13 @@ export class InvoiceService implements OnModuleInit, OnApplicationBootstrap {
         loggerCtx
       );
     });
-    logIfInvalidLicense(Logger, PLUGIN_NAME, loggerCtx, this.config.licenseKey);
+    //FIX ME
+    logIfInvalidLicense(
+      Logger as any,
+      PLUGIN_NAME,
+      loggerCtx,
+      this.config.licenseKey
+    );
```

Comment:
These can be removed, we will soon migrate this plugin to Vendure's marketplace and not use licens keys anymore

```suggestion
```